### ISSUE TITLE
why score_map have two channels?

### ISSUE THREAD

**Issue by fsluckymao** [tonghe90/textspotter#27]
Hi @tonghe90 : 
i have reviewed your train.pt file,  i found your score_map generated by 1*1 convoultion layer( "score_4s" in file) has two channels, as to your answer in  this issue:
https://github.com/tonghe90/textspotter/issues/16#issuecomment-405921502
its confuse me, should i perpare a correspoding two channels score map as surpervision information or just one channel ?

**Comment by tonghe90** [comment]
@fsluckymao The score map is text/non-text maps using softmax.

**Comment by fsluckymao** [comment]
@tonghe90 thank you for your reply, but is it means surpervision information need two channels?

**Comment by tonghe90** [comment]
@fsluckymao you can still make one channel ground truth and using the one hot encoding method to transform it to two channel
